resources/assets/js/components/JobBuilder/Tasks/JobTasks.tsx

```diff
@@ -129,29 +129,28 @@ const JobTasks: React.FunctionComponent<JobTasksProps &
           if (keyTask) {
             return {
               ...keyTask,
-              [locale]: {
-                description: task.description,
+              description: {
+                en: locale === "en" ? task.description : "",
+                fr: locale === "fr" ? task.description : "",
```

Comment:
This is setting the other locale value to null when saved. Try changing it to: `locale === "en" ? task.description : keyTask.description.en` (and same for french).